contrib/win32/openssh/sshd.vcxproj

```diff
@@ -1,473 +1,473 @@
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-bsdauth.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-krb5.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-options.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-pam.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-passwd.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-rhosts.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-shadow.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-sia.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth-skey.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-chall.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-gss.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-hostbased.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-kbdint.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-none.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-passwd.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2-pubkey.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)auth2.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)groupaccess.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)gss-serv-krb5.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)gss-serv.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)kexdhs.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)kexecdhs.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)loginrec.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)md5crypt.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)monitor.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)monitor_wrap.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)platform.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sandbox-capsicum.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sandbox-darwin.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sandbox-null.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sandbox-rlimit.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sandbox-seccomp-filter.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sandbox-systrace.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)servconf.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)serverloop.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)session.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sftp-common.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sshd.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)sshlogin.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\win32_sshpty.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\wmain_sshd.c" />
-    <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\spawn-ext.c" />
-  </ItemGroup>
-  <ItemGroup>
-    <ResourceCompile Include="version.rc" />
-  </ItemGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <ImportGroup Label="ExtensionTargets">
-  </ImportGroup>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Looks like CRLF issue.. please take care of this.

Reply:
done.